### PR TITLE
Avatar & alert component

### PR DESCRIPTION
**Pull request by Rishabh-078**
Avatar of different sizes and shapes along with different types of alert messages are added.

### CODE REVIEW

File: documentation.html [hunk 1]
```diff
@@ -43,7 +43,111 @@ <h1>Components</h1>
...
+                    <p><span class="avatar-size-info">avatar-xl</span> :10rem</p>
+                    <p><span class="avatar-size-info">avatar-l</span> :8rem</p>
+                    <p><span class="avatar-size-info">avatar-m</span> :6rem</p>
+                    <p><span class="avatar-size-info">avatar-s</span> :4rem</p>
+                </div>
+                <h5>Examples of Round Image Avatar:</h5>
+                <div class="container">
+                    <div class="avatar">
+                        <img class="ava-round-img avatar-xxl" src="./assets/avatar.png" alt="Avatar" /></div>
+                    <div class="avatar">
```
Comment (Ankur9669) [inline documentation.html:68]: The indentation could be improved

File: documentation.html [hunk 2]
```diff
@@ -43,7 +43,111 @@ <h1>Components</h1>
...
+                    style="width: 590px; height: 393px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
```
Comment (Ankur9669) [inline documentation.html]: The inline styling in iframe could be removed

**Review by Ankur9669 — APPROVED** [review]
Great Work!!!. The UI and the code are looking awesome. Do these changes and then you can merge it

**Comment by veereshbv04** [comment]
In front page add cursor:pointer for buttons

File: documentation.html (hunk 1 above)
Comment (Rishabh-078) [reply documentation.html:68]: Done

File: documentation.html (hunk 2 above)
Comment (Rishabh-078) [reply documentation.html]: Thanks for bringing it to my notice, I had forgotten about this.
Had made the required changes.

**Pull request merged**